### PR TITLE
Bug Fixes and additions

### PR DESCRIPTION
**Pull request by Dr-Gecko**
- Accomplishment page:
        Added this years CTF & "Various accomplishments", then that we hosted this years CTF
- Who we are page
      Reformatted so under the banner image it has a H2 heading "About", then added a H2 heading for our officers, added this years officers
- Resources
   fixed bug where Dr. Jang's site wasn't being properly shown in markdown order
   edited to properly reflect the fact that we have a CDC discord now
- Added a WiCyS dropdown menu, description is TBA however the club officers are listed

Issues fixed:
#44 (added accomplishments) 
#33 (updated copyright)

### CODE REVIEW

File: content/about.md [hunk 1]
```diff
@@ -3,15 +3,17 @@ title: About
 layout: 'infopage'
 ---
 
-The OSU Security Club was founded in 2014 by Daniel Reichert, and was guided by Dr. Yeongjin Jang until 2023. Currently, the club is advised by Zane Ma. We focus on gaining hands-on experience with exploiting vulnerabilities in systems and networks, defending against these attacks, and getting involved in the security community. There’s no experience or knowledge required, only an interest in learning more about this incredibly important and expanding field!
+## About
 
-Current leadership:
+The OSU Security Club was founded in 2014 by Daniel Reichert, and was guided by Dr. Yeongjin Jang until 2023. Currently, the club is advised by Dr. Zane Ma. We focus on gaining hands-on experience with exploiting vulnerabilities in systems and networks, defending against these attacks, and getting involved in the security community. There’s no experience or knowledge required, only an interest in learning more about this incredibly important and expanding field!
+
+## Current leadership:
```
Comment (cacama-valvata) [inline content/about.md]: can you make "about" and "current leadership" into #### headers? Want those way smaller, if theres a different way you think it makes sense to change those, feel free.

File: content/accomplishments.md [hunk 2]
```diff
@@ -3,10 +3,17 @@ title: Accomplishments
 layout: 'infopage'
 ---
 
-[CTFtime Profile](https://ctftime.org/team/12858)
+
+## Various Acomplishments
```
Comment (cacama-valvata) [inline content/accomplishments.md]: i have very conflicted feelings about this year's ctf so i'd rather just include that we won it down below and remove this section entirely. i like the moving of the ctftime link underneath the header.

i will update you once i have the public bsidespdx ctf repo made, so can you put in an issue about replacing that link once we merge this PR?

File: content/resources.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ Also join the @CTF role on Discord to participate in CTF with us!
     - To join, see #announcements or message an officer on Discord.
 - CTF101: [https://ctf101.org/](https://ctf101.org/)
 - TrailOfBits CTF Field Guide: [https://trailofbits.github.io/ctf/](https://trailofbits.github.io/ctf/)
-- CS499: Cyber Attacks And Defense (taught by our awesome advisor, Dr. Jang, check out the Lectures at this link): [https:// cand.unexploitable.systems/cal.html](https:// cand.unexploitable.systems/cal.html)
+- CS499: Cyber Attacks And Defense (taught by our awesome advisor, Dr. Jang, check out the Lectures at this link): [https://cand.unexploitable.systems/cal.html](https://cand.unexploitable.systems/cal.html)
```
Comment (cacama-valvata) [inline content/resources.md:30]: thanks for this edit.

can you make an issue that we need to update our resources page?

File: content/resources.md [hunk 4]
```diff
@@ -111,11 +111,10 @@ Materials relating to …. well… malware reversing, forensics, threat intellig
 
 **CDC Materials**
 
-Materials for learning Cyber Defense Competition skills.
-Join us and learn about CDC in the #cdc channel by adding the CDC role in Discord!
+Materials for learning Cyber Defense Competition Skills.
```
Comment (cacama-valvata) [inline content/resources.md]: pls revert both of these lines:
- skills doesnt need to be caps
- that discord is currently not a good place for resources and we need to rethink how we're linking our discords anyhow

File: content/wicys.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+---
+title: WiCyS @ OSU
+layout: 'infopage'
+---
+
+## About WiCyS @ OSU
+TBA
+
+## Current leadership:
```
Comment (cacama-valvata) [inline content/wicys.md]: same edit as on the about.md, can we turn these into `####`

**Review by cacama-valvata — CHANGES_REQUESTED** [review]

**Comment by cacama-valvata** [comment]
Also, this doesnt fully close out issue #44. We can either keep the issue open or have that completed in this PR.

File: content/wicys.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (cacama-valvata) [inline content/wicys.md:1]: additionally, lets find a good picture for their banner/background on the page -- ask the wicys folks?

**Review by cacama-valvata — CHANGES_REQUESTED** [review]

File: content/wicys.md (hunk 6 above)
Comment (Dr-Gecko) [reply content/wicys.md:1]: Creating issue for this

**Review by cacama-valvata — APPROVED** [review]
Looks great!

**Pull request merged**